Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -113,7 +113,11 @@ def validater(username, access_token):
 
         def prompt():
             result = "GitHub's API\nProvide {} username and access token to create and update pull requests".format(hostname)
-            if webkitcorepy.Terminal.open_url('{}?scopes=repo,workflow&description={}%20Local%20Automation'.format(token_url, self.name)):
+            if webkitcorepy.Terminal.open_url(
+                '{}?scopes=repo,workflow&description={}%20Local%20Automation'.format(token_url, self.name),
+                prompt='Please press ENTER to open the GitHub token generation web page.\n'
```

Comment:
I'm not sure why you insist on upper-casing ENTER, but I also just remembered/noticed that it's marked `return` on my keyboard, and there is no Enter key on it.